### DISCUSSION TITLE
Worth Paying Extension Feature

### DISCUSSION THREAD

**Discussion by taufik-nurrohman** [mecha-cms/mecha#215] [Polls]
Comment below if you have more than 1 option to choose or have other options to consider!

**Comment by lokersumatera** [comment]
- Database management system.
- Member management system.
- Search engine optimization.
- Third-party integration (you pay for the maintenance).
- Complexity.

**Reply by lokersumatera** [reply]
kayaknya semuanya bisa di bikin fitur berbayar kang.

**Reply by taufik-nurrohman** [reply]
Okey. Silakan pilih salah satu yang mau dijadikan prioritas.

**Reply by yuliakbard** [reply]
SEO dulu lah kang. hehehe

**Reply by yuliakbard** [reply]
kang, request next nya, kode editor nya dicanggihin lagi kang, saat save gak naik ke atas melulu. lumayan repot kalo edit kode dari kode editor mecha macam blogger

**Reply by yuliakbard** [reply]
insight/statistic halaman juga masih belum ada di mecha. semoga juga ada extension nya

**Reply by taufik-nurrohman** [reply]
> kang, request next nya, kode editor nya dicanggihin lagi kang, saat save gak naik ke atas melulu. lumayan repot kalo edit kode dari kode editor mecha macam blogger 

Itu efek natif dari [F3H](https://github.com/taufik-nurrohman/f3h) dan web normal. Walau pakai AJAX tapi proses muat tidak dikontrol secara individu di formulir, makannya kesannya seperti tanpa AJAX, cuma efek _loading_ saja yang hilang.

**Reply by taufik-nurrohman** [reply]
> insight/statistic halaman juga masih belum ada di mecha. semoga juga ada extension nya

Pakai basis data paling. Kalau pakai flat-file masih bisa sebenarnya, misal dengan cara nyimpen data &ldquo;hits&rdquo; ke hirarki `.\lot\statistic\:year\:month\:day\view.data` cuma dari pengalaman pribadi, kalau data sudah banyak jadi susah/lelet untuk dipindah-pindah/dihapus. Kalau untuk membaca berkas tidak ada masalah.

**Reply by yuliakbard** [reply]
automation.

sepertinya saya butuh automation terutama untuk gambar. up up

**Reply by taufik-nurrohman** [reply]
Image resize? Udah bawaan di ekstensi `panel.image`.
